style/style.css

```diff
@@ -7,308 +7,519 @@ body {
-    padding: 12px;
-    margin: 20px;
-    color: white;
-    font-weight: 500;
-    font-size: 17px;
-    text-align: center;
-    background: #36b37f;
-    border: #36b37f;
-    border-radius: 4px;
+    margin: 24px 12.3em;
   }
 
   .card-info {
     display: grid;
-    grid-row-gap: 20px;
+    margin: 50px 200px;
+    grid-template-columns: 33.33% 33.33% 33.33%;
+    grid-row-gap: 24px;
+    grid-column-gap: 10px;
+    justify-content: center;
   }
 
-  .card {
-    display: flex;
-    align-items: center;
-    flex-direction: column;
-    margin-left: 24px;
-    margin-right: 24px;
-    background: #ebf0ee;
-    border: 1px solid #d0d9d4;
-    border-radius: 8px;
+  .main4-contact {
+    position: relative;
   }
 
-  .card img {
-    margin: 20px 1px;
-    padding: 20px;
+  .main4 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0 211px;
   }
 
-  .card h1 {
-    font-weight: 600;
-    font-size: 24px;
-    color: #344563;
-    margin: 20px 1px;
+  .main4 h1 {
+    flex: 1 0.3em;
+    margin-right: 1em;
+    text-align: center;
+    font-size: 1.6em;
+    line-height: 52px;
   }
 
-  .card ul {
+  .main4 form {
+    flex-grow: 1.5;
+    flex-shrink: 1;
+    flex-basis: 0.7em;
     display: flex;
-    justify-content: space-around;
-    flex-wrap: wrap;
-    margin-left: 15px;
-    margin-right: 15px;
-    margin-bottom: 40px;
-  }
-
-  .main4 {
-    margin: 24px;
-    margin-top: 60px;
+    justify-content: flex-start;
   }
 
-  .main4 h1 {
-    font-size: 40px;
-    font-weight: 700;
-    line-height: 52px;
-    color: #172b4d;
-    text-align: center;
+  .main4 form button {
+    align-self: flex-start;
   }
 
-  .contact-form {
-    margin-top: 58px;
+  .bg-image {
     display: flex;
-    flex-direction: column;
   }
 
-  .contact-form input {
-    margin: 10px 0;
-    padding: 15px 16px;
-    border: 1px solid #d0d9d4;
-    box-sizing: border-box;
-    border-radius: 4px;
+  .bg-image-1 {
+    position: absolute;
+    background-image: url("/image/bg-image-1.png");
+    height: 261.09px;
+    width: 236px;
+    background-repeat: no-repeat;
+    background-size: contain;
+    left: 0;
+    bottom: 0;
+    top: 18em;
   }
 
-  .contact-form textarea {
-    font-family: "Inter", sans-serif;
-    height: 85px;
-    padding: 12px 10px;
-    margin: 10px 0;
-    margin-bottom: 24px;
-    border: 1px solid #60c095;
+  .bg-image-2 {
+    position: absolute;
+    background-image: url("/image/bg-image-2.png");
+    background-repeat: no-repeat;
+    width: 130.28px;
+    height: 105.85px;
+    background-size: contain;
+    left: 15em;
   }
 
-  .contact-form button {
-    font-family: "Inter", sans-serif;
-    background: #36b37f;
-    border: #36b37f;
-    width: 130px;
-    color: white;
-    font-size: 17px;
-    font-weight: 500;
-    padding: 12px;
-    align-self: center;
-    border-radius: 4px;
+  .bg-image-3 {
+    position: absolute;
+    background-image: url("/image/bg-image-3.png");
```

Comment:
- [ ] Here you have the same bug pointed above, so kindly try to solve this issue before submitting your project for an other review 🙏🏼

Reply:
### Fixed🚀